src/qoi/include/grins/qoi_base.h

```diff
@@ -78,7 +78,9 @@ namespace GRINS
      * Method to allow QoI to cache any system information needed for QoI calculation,
      * for example, solution variable indices.
      */
-    virtual void init( const GetPot& input, const MultiphysicsSystem& system );
+    virtual void init( const GetPot& input,
+                       const MultiphysicsSystem& system,
+                       unsigned int qoi_num );
```

Comment:
It's internal, but it's already led to internal bugs in this branch.  Thanks @dmcdougall for bringing it to my attention!